src/Entity/Query/Sparql/SparqlArg.php

```diff
@@ -16,6 +16,21 @@
  */
 class SparqlArg {
 
+  /**
+   * The variable separator.
+   *
+   * Normally, a parameter like "field_name.column" can be passed in the
+   * condition indicating that the property "column" should be checked for that
+   * field. This is fine for standard SQL databases but for SPARQL, we cannot
+   * simply concatenate the field an column with an underscore as that could
+   * conflict in fields like field_a_b.c and field_a.b_c.
+   *
+   * SPARQL accepts some unicode characters, one of which is the middle dot.
```

Comment:
Let's add also the link https://www.w3.org/TR/sparql11-query/#rVARNAME for better docs.